spk/dnscrypt-proxy/src/service-setup.sh

```diff
@@ -1,15 +1,15 @@
 SVC_CWD="${SYNOPKG_PKGDEST}"
 DNSCRYPT_PROXY=${SYNOPKG_PKGDEST}/bin/dnscrypt-proxy
 PID_FILE=${SYNOPKG_PKGDEST}/var/dnscrypt-proxy.pid
-CFG_FILE="${SYNOPKG_PKGDEST}/var/dnscrypt-proxy.toml"
```

Comment:
Please let configuration file in "var"  which is the only location to survive to upgrade process "by default".

Reply:
Interesting. I always put configs in [/etc](https://refspecs.linuxfoundation.org/FHS_3.0/fhs-3.0.html#purpose6). Generally  [/var](https://refspecs.linuxfoundation.org/FHS_3.0/fhs-3.0.html#purpose31) is for data that changes while running the software. Ok I'll change it back.